app/src/main/java/org/wikipedia/settings/Prefs.java

```diff
@@ -875,6 +875,13 @@ public static boolean showEditActionTranslateDescriptionsUnlockedDialog() {
     public static void setShowEditActionTranslateDescriptionsUnlockedDialog(boolean enabled) {
         setBoolean(R.string.preference_key_show_edit_action_translate_descriptions_unlock_dialog, enabled);
     }
+    public static boolean showMultilingualTask() {
```

Comment:
does the `showMultilingualTask()` actually being used?

Reply:
Done